### PR TITLE
Fix rspec scaffolding script after the SonarAnalysisContext refactoring

### PR DESCRIPTION
**Pull request by gregory-paidis-sonarsource**
hehe :)

### CODE REVIEW

File: scripts/rspec/rspec-templates/Rule.CS.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ public sealed class $DiagnosticClassName$ : SonarDiagnosticAnalyzer
     public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(Rule);
 
     protected override void Initialize(SonarAnalysisContext context) =>
-        context.RegisterSyntaxNodeActionInNonGenerated(c =>
+        context.RegisterNodeAction(c =>
```
[the remark this answers is not in the document]
Comment (gregory-paidis-sonarsource) [reply scripts/rspec/rspec-templates/Rule.CS.cs:34]: Done!

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM, -18 LOC is good enough :D

**Comment by gregory-paidis-sonarsource** [comment]
> LGTM, -18 LOC is good enough :D

hey @pavel-mikula-sonarsource , I forgot some usings, please review the last commit once again :)

**Review by pavel-mikula-sonarsource — APPROVED** [review]
Still LGTM, can't say no to removing more code

**Pull request merged**